### PR TITLE
Fix: `drom project -f; make opam-cross`

### PR DESCRIPTION
**Pull request by nberth**
Followup of #272: update project and cross-compilation files.

### CODE REVIEW

File: opam/sql_parser.opam [hunk 1]
```diff
@@ -55,7 +55,7 @@ depends: [
   "cobol_parser" {= version}
   "cobol_common" {= version}
   "ppx_deriving" {>= "5.2.1"}
-  "menhir" {= "20230415"}
+  "menhir" {>= "20230415"}
```
Comment (emilienlemaire) [inline opam/sql_parser.opam:58]: There seem to be a mismatch with `src/lsp/sql_parser/package.toml:71`

File: src/lsp/sql_parser/package.toml [hunk 2]
```diff
@@ -68,7 +68,7 @@ version = ">=1.2"
 # package tools dependencies
 [tools]
 ppx_deriving = ">=5.2.1"
-menhir = ">=20230415"
+menhir = "20230415"
```
Comment (emilienlemaire) [inline src/lsp/sql_parser/package.toml:71]: ditto

**Review by emilienlemaire — APPROVED** [review]
Ok to merge after checking my concern

**Review by ddeclerck — APPROVED** [review]

File: src/lsp/sql_parser/package.toml (hunk 2 above)
Comment (nberth) [reply src/lsp/sql_parser/package.toml:71]: A version without a relational operator in a `package.toml` is interpreted as a lower-bound by `drom`. The `>=` is optional (but I can put it back anyways).

**Pull request merged**